sound/sequences/00_sound_player.s

```diff
@@ -2345,8 +2349,9 @@ chan_setinstr 7
 chan_setlayer 0, .layer_DCA
 chan_end
 
+// layer_portamento_voice and layer_note1_voice intentionally not used
```

Comment:
why not?

Reply:
Same reason as above; the lack of variation sounds worse in my opinion. Though it's much less extreme in this case so I see it being less likely to bother people.